### ISSUE TITLE
Partition Corrupts after a certain number of songs

### ISSUE THREAD

**Issue by darthchewy** [Dimencia/LuteBot3#20]
The partition corrupts after a certain number of songs, Same problem with lutemod 3.3.3 and the latest beta 3.4.0(beta6). I currently have 195 songs and it seems like it always corrupts songs #174 and #175 Doesn't matter what the song is named or what song is in those spots. When you put all the songs in the partition the first time everything plays fine once. But subsequent loading of lutebut will corrupt those song spots everytime messing up the names and then they won't load in game.

**Comment by Dimencia** [comment]
This is a 'known' issue that I still haven't gotten around to fixing because at 250 partitions I still haven't experienced it, so it's hard to figure out why it happens.  A workaround is to move or rename your PartitionIndex[0] file when it reaches that point, and let it create a new one, then swap their names anytime you want to swap between the two 'playlists' you've created

Leaving the issue open because I still need to try to find out why, though

**Comment by darthchewy** [comment]
This might be a stupid response but I think it might have something to do with name lengths. I've noticed as I've moved around and deleted songs that the partition doesnt actually corrupt on how many songs there are but the length of the titles of the songs/band names and meaning not the individual tracks adding up but just the overall accumilation of letters. Basically the longer the band names and song names, it all adds up over time and then corrupts depending on the if you have alot of songs with long band and song names. Just a idea. hopefully this response isnt dumb, just trying to help. I absolutely love your program and either way thanks for creating this. I'm a musician and I love seeing what this program is capable of. 

 
 

Sent: Saturday, June 18, 2022 at 4:29 PM
